.gitignore

```diff
@@ -6,3 +6,5 @@ dependency-reduced-pom.xml
 .classpath
 .project
 .settings/
+.DS_Store
+gradoop-flink/src/test/resources/data/csv/input_indexed/
```

Comment:
it's this line that's causing the test to fail because this is where it get's it data from